scripts/mlow-vectors/mlow_frames.c

```
@@ -0,0 +1,153 @@
+    int err = 0;
+    OpusEncoder *enc = opus_encoder_create(FS, 1, OPUS_APPLICATION_VOIP, &err);
+    if (err != OPUS_OK || !enc) {
+        fprintf(stderr, "error: encoder create: %s\n", opus_strerror(err));
+        return 1;
+    }
+    opus_encoder_ctl(enc, OPUS_SET_USING_SMPL(1));
+    opus_encoder_ctl(enc, OPUS_SET_BITRATE(bitrate));
+    opus_encoder_ctl(enc, OPUS_SET_FORCE_CHANNELS(1));
+
+    OpusDecoder *dec = opus_decoder_create(FS, 1, &err);
+    if (err != OPUS_OK || !dec) {
```

Clang (14.0.6)

[note] +1, including nesting penalty of 0, nesting level increased to 1

(clang)